packages/engine/Source/Scene/Model/ModelRuntimePrimitive.js

```diff
@@ -199,7 +199,8 @@ ModelRuntimePrimitive.prototype.configurePipeline = function (frameState) {
   const mode = frameState.mode;
   const use2D =
     mode !== SceneMode.SCENE3D && !frameState.scene3DOnly && model._projectTo2D;
-  const exaggerateTerrain = frameState.verticalExaggeration !== 1.0;
+  const exaggerateTerrain =
```

Comment:
Can we rename this variable to `exaggerationOn` or something like that? The leftover "terrain" naming looks like an oversight from the previous change to `verticalExaggeration`

Reply:
complete